==== triton/test/Analysis/test-alias.mlir#5 - /google/src/cloud/shyshkov/mlir_8dbddb17180fff0ed881d75689651769b9a9b483_1690193084/triton/test/Analysis/test-alias.mlir ====
# action=edit type=text
--- triton/test/Analysis/test-alias.mlir	2023-05-26 16:07:06.000000000 -0700
+++ triton/test/Analysis/test-alias.mlir	2023-07-24 03:52:39.000000000 -0700
@@ -192,10 +192,10 @@
   // CHECK-NEXT: %arg9 -> %cst_1
   // CHECK-NEXT: %0#0 -> %cst
   // CHECK-NEXT: %0#1 -> %cst_0
-  // CHECK-NEXT: %0#2 -> %cst_2,%cst_2
+  // CHECK-NEXT: %0#2 -> %cst_1,%cst_2,%cst_2
   %a_shared, %b_shared, %c_shared = scf.for %iv = %lb to %ub step %step iter_args(%a_shared = %a_shared_init, %b_shared = %b_shared_init, %c_shared = %c_shared_init) -> (tensor<128x32xf16, #A_SHARED>, tensor<128x32xf16, #A_SHARED>, tensor<128x32xf16, #A_SHARED>) {
     // CHECK-NEXT: %arg11 -> %cst_1,%cst_2,%cst_2
-    // CHECK-NEXT: %1 -> %cst_2,%cst_2
+    // CHECK-NEXT: %1 -> %cst_1,%cst_2,%cst_2
     %c_shared_next = scf.for %jv = %lb to %ub step %step iter_args(%c_shared_next = %c_shared) -> (tensor<128x32xf16, #A_SHARED>) {
       // CHECK-NEXT: %2 -> %cst_2,%cst_2
       %c_shared_next_next = scf.if %i1 -> tensor<128x32xf16, #A_SHARED> {
